Use a list of scripts to add script tags to page

andyg0808 opened the pull request:
This avoids the need to change the `pdf.js` project every time we want to add another script to this page.

QA
---
We'll QA this as part of the pull that uses it.
qa_req 0

Commit:
Use a list of scripts to add script tags to page

This avoids the need to change the `pdf.js` project every time we want
to add another script to this page.

djmetzle commented:
Nice! CR :gear:

emmalopezcode commented:
yay! cutting back on tech debt! CR 🎈

Commit:
Restrict travis to most recent node which works with Gulp

It seems that newer versions of node crash Gulp. Updating Gulp to latest
seems non-trivial. Upstream already uses a newer version of Gulp, and we
should probably pull that work in if we want to use a newer Gulp.

djmetzle commented:
Nice! Good job fixing CI too @andyg0808! A+

CR :+1: :+1:

emmalopezcode commented:
CR 🎈

danielcliu commented on .travis.yml:
```diff
@@ -7,7 +7,7 @@ git:
 language: node_js
 node_js:
-  - node
+  - 10
```
Is there any reason for choosing 10 instead of a more recent version?

djmetzle replied:
12 would get us ES2017 features :raised_eyebrow:

andyg0808 replied:
Newer versions of node crash Gulp, as far as I found. Updating Gulp to latest seems non-trivial. Upstream already uses a newer version of Gulp, and we should probably pull that work in if we want to use a newer Gulp.

ardelato commented:
deploy_block :question: on addressing @danielcliu 's question https://github.com/iFixit/pdf.js/pull/22#discussion_r562100450

andyg0808 commented:
un_deploy_block :+1: 
Answered question.

evannoronha commented:
CR :fire:

andyg0808 commented:
dev_block :+1: There seems to be a collision on the variable name here.

Commit:
Use more unique variable name

It appears that we had a collision on the `$scripts` variable name, so
we weren't including the right scripts on the page.

andyg0808 commented:
un_dev_block :fish:
Changed the variable to something more unique.

evannoronha commented:
CR 🔥

danielcliu commented:
CR ✌️

djmetzle commented:
CR :page_facing_up:
